wiki/Help_centre/Client/de.md

```diff
@@ -0,0 +1,521 @@
+
+Die einzig wahre Lösung ist der Kauf einer Tastatur, die in diesem Fall mehr gleichzeitige Tastendrücke unterstützt. Du kannst nach einer Funktion mit dem Namen *n-key rollover* (oder *NKRO*) schauen. NKRO-Tastaturen haben zum größten Teil keine Probleme mit Ghosting, aber sind normalerweise teuer.
+
+### Mein Cursor spielt verrückt, sobald ich versuche, osu! zu spielen
+
+**Die häufigste Ursache dafür ist, dass die Mausempfindlichkeit im Spiel zu hoch eingestellt ist.**
+
+Sprunghafte, unregelmäßige Bewegungen sind ein Paradebeispiel dafür, vor allem wenn du ein Tablet benutzt.
+
+Das Setzen der Empfindlichkeit im Spiel auf 1.0x (was der Standard ist), wird das in den meisten Fällen lösen, aber kann dazu führen, dass sich dein Cursor träge anfühlt. Wir empfehlen, `Raw Input` in den Spieloptionen zu aktivieren und die Treiber deiner Peripheriegeräte zu benutzen, um die Empfindlichkeit zu verändern.
+
+Du kannst auch die *DPI* deiner Maus anpassen, falls du eine verwendest. Höhere DPI-Einstellungen bieten eine akkuratere Positionierung für den Spiel-Client und können eventuelle Jitter ausgleichen.
+
+Wir schlagen ebenso vor, dass du sicherstellst, dass deine Peripheriegeräte sauber und frei von Schmutz oder Ablagerungen sind. Ein Haar oder ein bisschen Staub vor dem optischen Sensor einer Maus kann sehr seltsame Dinge mit der Bewegung des Cursors anstellen.
+
+#### Raw Input aktivieren
+
+**Raw Input zwingt das Spiel dazu, eine sauberere und direktere Methode für den Zugriff auf die Informationen deiner Peripheriegeräte zu verwenden. Das erhöht allgemein die Genauigkeit der Ausgabe des Geräts.**
+
+Um Raw Input zu aktivieren, folge diesen Schritten:
+
+1. Öffne osu!.
+2. Klicke auf den Button `Options` im Hauptmenü oder drücke `Strg` + `O`.
+3. Tippe `Raw` in die Schnellsuche, um direkt zu der Einstellung zu gelangen.
+4. Klicke auf die Einstellung `Raw Input`, um sie zu aktivieren.
+
+### Ich spiele mit einem Tablet und mein Cursor ist zu langsam!
+
+**Du kannst dieses Problem lösen, indem du einen kleineren Tablet-Bereich für dein Gerät festlegst.**
+
+Bei den meisten Wacom-Tablets wird das via dem Tab `Mapping` der Applikation `Wacom Tablet Properties` erledigt.
+
+![Anzeige von Wacom Tablet Properties, die die Mapping-Einstellungen anzeigt](img/tablet-area.png "Benutzeroberfläche, um den Tablet-Bereich auf Wacom-Geräten zu ändern")
+
+Du musst den Hersteller deines Tablets fragen, um herauszufinden, wie der Tablet-Bereich für dein spezifisches Peripheriegerät eingestellt werden kann.
+
+Alle Tablets der Marke osu! unterstützen Raw Input standardmäßig und die Empfindlichkeit kann dort leicht sowie spontan durch die Einstellung  Raw Input im Spiel und den Schieberegler für die Mausempfindlichkeit angepasst werden.
+
+### Ich erlebe Lag-Spitzen oder Screen Tearing während dem Spielen!
```

Comment:
```suggestion
### Ich erlebe Lag-Spitzen oder Stottern während dem Spielen!
```
Screen Tearing is a bit different from stuttering so maybe this just works

Reply:
Never really used `Stottern` in the context of computer monitors, so that's why I used this originally. It's fine though.